framework/src/actions/AddPeriodicBCAction.C

```diff
@@ -152,6 +158,8 @@ AddPeriodicBCAction::act()
     _problem->addGhostedBoundary(p.pairedboundary);
 
     nl.dofMap().add_periodic_boundary(p);
+    if (displaced_problem != NULL)
```

Comment:
Like` if(displaced_problem)`?  What I am using here is the same as anywhere else. Honestly, I paste it from `FEProblem` :)